tests/console_test.go

```diff
@@ -170,33 +170,22 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 		})
 
 		Context("without a serial console", func() {
-			var vmi *v1.VirtualMachineInstance
 
-			BeforeEach(func() {
-				vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+			It("[test_id:4116]should create the vmi without any issue", func() {
+				vmi := libvmi.NewAlpine()
 				f := false
 				vmi.Spec.Domain.Devices.AutoattachSerialConsole = &f
-			})
-
-			It("[test_id:4116]should create the vmi without any issue", func() {
-				tests.RunVMIAndExpectLaunch(vmi, 30)
-			})
-
-			It("[test_id:4117]should not have the  serial console in xml", func() {
```

Comment:
We could have a debate if this is indeed the right place to test it and if needed at all.

Reply:
Right. For KubeVirt users, xml is an implementation detail. IMHO we should have a very good reason to mention it in a functional test. I feel that too many tests use `util.GetRunningVMIDomainSpec` but I'd leave tackling this for another PR.

Reply:
Agree :+1: Maybe we can open an issue so we don't forget to re-think it.